src/main/java/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -1,23 +1,35 @@
 package org.usfirst.frc4904.robot;
 
 import org.usfirst.frc4904.standard.custom.controllers.CustomXbox;
-
+import org.usfirst.frc4904.standard.custom.motioncontrollers.CANTalonSRX;
+import org.usfirst.frc4904.robot.subsystems.ProngIO;
+import org.usfirst.frc4904.standard.subsystems.motor.Motor;
+import org.usfirst.frc4904.standard.custom.PCMPort;
 
 public class RobotMap {
     public static class Port {
         public static class HumanInput {
             public static final int xboxController = 1;
 
         }
-        public static class CANMotor {}
+        public static class CANMotor {
+            //Define CANMotor.prongIOExpanderMotor with a correct port.
```

Comment:
For Todos, add them on the line of the port, with `//TODO: Define with a correct port.`